releases.md

```diff
@@ -100,6 +100,12 @@ deprecation window timers begin from when the release is published.
 
 ## Current Releases
 
+### v0.22 (LTS)
+- **Latest Release**: [v0.22.0][v0-22-0] (2024-08-19)
+- **Initial Release**: [v0.22.0][v0-22-0] (2024-08-19)
+
+[v0-22-0]: https://github.com/tektoncd/chains/releases/tag/v0.22.0
+
```

Comment:
Do you need to remove v0.19 (LTS) from support?

Reply:
I don't think we can based on our [stability levels](https://github.com/tektoncd/chains/blob/821f7c446131a9cb303a8a9f63ced6ba17ca37fc/releases.md?plain=1#L57).